R/read_alevin.R

```diff
@@ -199,6 +208,8 @@ read_alevin_metadata <- function(quant_dir){
   meta$af_resolution <- quant_info[['resolution_strategy']]
   meta$af_tx2gene <- cmd_info[['tgMap']]
   meta$usa_mode <- quant_info[['usa_mode']]
+  meta$num_cells <- quant_info[['num_quantified_cells']]
+  meta$version_10X <- names(cmd_info)[[3]]
```

Comment:
So this is something I was struggling with. I forgot to mention this in my previous comment, but I agree that this isn't the most stable method of getting the 10X version moving forward. This was the only way I could get it using the alevin output which is all we have currently as input to the `read_alevin` function where the metadata is added to the sce. Moving forward, I do think the best place to get it will actually be from the `scpca-library-metadata.tsv` sheet that we have that becomes the metadata map in the nextflow workflow. Because we plan on creating the QC report from the workflow, another option is to actually use the 10X version as input to the function along with the sample ID? Unless you have other ideas on how to grab this information I am open to suggestions.

Reply:
We could make it an optional argument to `read_alevin`... I think there are a number of places that it could go, but that probably makes the most sense, so it ends up in the SCE metadata if desired.